setup.cfg

```diff
@@ -63,14 +63,10 @@ test =
     flake8>=4.0.1
     mypy
     pydocstyle
-    pylint
-    pylint-fixme-info
-    pylint-pytest>=1.1.3
     pytest
     pytest-check>=2.0
     pytest-mock
     pytest-subprocess
-    ruff==0.3.5
```

Comment:
can you please, on a separate commit, re-generate the requirements-* files?

Reply:
A separate commit or a separate PR?

Reply:
https://github.com/canonical/rockcraft/pull/678

Reply:
I meant a separate commit but I don't remember why I asked that 😆 
I think putting that change on this PR would be fine and more correct